btrack/_localization.py

```diff
@@ -147,32 +131,27 @@ def segmentation_to_objects(
     scale: Optional[Tuple[float]] = None,
     use_weighted_centroid: bool = True,
     assign_class_ID: bool = False,
-) -> list:
-    """Convert segmentation to a set of btrack.PyTrackObject.
+) -> List[btypes.PyTrackObject]:
```

Comment:
In case you're not aware from `python>=3.9` we can just do `list[btypes.PyTrackObject]`, but not backwards compatible without `from __future__ import annotations`

Reply:
Unfortunately, I've since realised my understanding of `from __future__ import annotations` was incorrect. But the first bit will be true whenever `3.8` is no longer officially supported